testing-guidance/workflow-tests/scripting/script-canvas-tests.md

```diff
@@ -60,54 +64,72 @@ Responsive Script Canvas editor with all 5 basic tools enabled. No Asset Process
+| Nodes behavior           | <ol><li>Add nodes with various data value types to the graph and modify their values.</li><li>Attempt to connect two different value type pins.</li><li>Add input and operand to the node (e. g. Print/Add (+) nodes).</li><li>Hide/show unused slots on the node.</li><li>Convert value to reference and reference to value by clicking RMB on data pin.</li><li>Disable and enable nodes.</li><li>Insert long string into node string value field to extend the node.</li></ol> | <ul><li>If data values cannot be modified.</li><li>If nodes with long strings are visually distorted.</li><li>If disabled nodes are not grayed out.</li> |
+| Connections behavior                  | <ol><li>Create a connection between nodes by dragging connection from output pin onto input pin.</li><li>Drag a connection over empty space in graph and release LMB, in context menu search for desired node to be created and connected to.<li>Connect two nodes by using Node Coupling feature, mentioned feature is enabled by default in Global Preferences. Hover one node over another and hold it above until nodes connect.</li><li>Delete connection by selecting the connection and pressing Del key.</li><li>Delete connection by dragging one node and shaking it, shaking de-splice feature is enabled by default.</li><li>Delete connection by holding Alt key and clicking LMB on connection.</li></ol> | <ul><li>If connection is not created.</li><li>If node is not created after selecting it from context menu.</li><li>If connection cannot be deleted.</li> |
+---
 
-## Area: Advanced Script Canvas graph Tests
+### Area: Advanced Script Canvas graph Tests
 Tests in this verify more advanced core features of script canvas.
 
-### Project Requirements
-Any project with a new level. SC editor with the 5 basic tools enabled
+**Project Requirements** 
+Any project with a new level. SC editor with the 5 basic tools enabled.
 
-### References
-Main Script Canvas editor view with advanced control buttons highlighted<br>
-![SC graph view and basic controls](images/SCeditor_graphview_controls.png)<br>
+**References**
+Main Script Canvas editor view with advanced control buttons highlighted.
+[![SC graph view and basic controls](https://github.com/o3de/sig-content/raw/main/testing-guidance/workflow-tests/scripting/images/SCeditor_graphview_controls.png)](https://github.com/o3de/sig-content/blob/main/testing-guidance/workflow-tests/scripting/images/SCeditor_graphview_controls.png)
```

Comment:
Let's listify this so that the image renders offset and doesn't take away from the flow of the document.

You can also play with the formatting of this with [collapsed sections](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections).